zeppelin-display/src/main/scala/org/apache/zeppelin/display/angular/AbstractAngularElem.scala

```diff
@@ -131,8 +131,8 @@ abstract class AbstractAngularElem(val interpreterContext: InterpreterContext,
     // create AngularFunction in current paragraph
     val functionName = eventName.replaceAll("-", "_") + "_" + uniqueId
     val elem = this % Attribute(None, eventName,
-      Text(s"${functionName}=$$event.timeStamp"),
-      Null)
+      Text(s"${functionName}=${functionName} + 1"),
+      Null) % Attribute(None, "ng-init", Text(s"${functionName} = 0"), Null)
```

Comment:
Looks like initializing value using ng-init calls `callback` function once, before the actual event.
I think the `${functionName}` need to be initialized in the following line

```
val angularObject = addAngularObject(functionName, 1)
```